README.md

```diff
@@ -65,15 +65,27 @@ Create your enviroment variables based on the examples of ```.env.example```
 
 After copying the examples, make sure to fill the variables with new values.
 
+# :runner: Getting Started
+
+**Running with docker-compose**
+
+To prepare database and run the application, you just need execute docker-compose after set your environment (.env file):
+
+```docker-compose up```
+
+With this command, the port `3000` will be available to access the application.
+
 **Setup a database**
 
+But if you preffer, you can setup your database with docker compose, and the application manually.
+
 Install [Postgres](https://www.postgresql.org/) to create a database or if you have [Docker](https://www.docker.com/) in your machine, fill the environment values related to database configurations and the run the following commands in order to create a postgres container.
 
-```docker-compose up```
+```docker-compose up postgres```
 
-Execute the [Database Queries](https://github.com/LauraBeatris/foodfy/blob/master/src/database/seeds.js) to create tables, relationships and procedures.
+Execute the follow command to create tables, relationships and procedures:
 
-# :runner: Getting Started
```

Comment:
Why remove the ``Getting Started`` section? I think that we should put the instruction related to creating table, relationships, procedures, and running seeds, into that section.

Reply:
I didn't remove this section, I just move the *setup database* for inside. Before, the setup of database were inside the *Installation section*, I think that make more sense to be inside *Getting start* section. What do u think?

Reply:
Well...
Thinking about it now, that makes sense :smile: